tests/DevApps/AzureFunctions/Function1.cs

```diff
@@ -0,0 +1,24 @@
+using Microsoft.AspNetCore.Http;
```

Comment:
when I pulled it locally, I had a SampleFunc folder showing up, which was messing things up:
![image](https://github.com/AzureAD/microsoft-identity-web/assets/19942418/84cc3b4c-46cd-438f-9676-3aedc05977e6)

so i deleted it and things seem to be building now. I don't see it in any of your files though... 🤔